### PR TITLE
Created home page

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/108062/google-home-page

### CODE REVIEW

File: src/components/Home/__tests__/Home.test.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import { cleanup, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Home } from "../Home";
+import { render } from "../../../../testing";
+
+const events = [{
+  "id": "tb34kdqm2cr96rvk2blhprpgks_20230503T083000Z",
```
Comment (iTweeno) [inline src/components/Home/__tests__/Home.test.tsx:7]: uhh dunno if we want the link for our standups here ahah, just add other characters or something

File: src/utils/__tests__/getMapFilteredEventsByDay.test.ts [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+            "dateTime": "2021-02-16T19:00:00+02:00",
+            "timeZone": "Europe/Kiev"
+          },
+          "calendarId": "usi9q4gc0eln2v7392tqb8k4k4@group.calendar.google.com",
+          "calendarSummary": "Activities"
+        },
+        {
+          "id": "coqjidpncpi32bb560qjcb9k71j38bb2clj3eb9j6ko64c3170r3epb56k",
+          "summary": "Психотерапевт",
+          "htmlLink": "https://www.google.com/calendar/event?eid=Y29xamlkcG5jcGkzMmJiNTYwcWpjYjlrNzFqMzhiYjJjbGozZWI5ajZrbzY0YzMxNzByM2VwYjU2a18yMDIxMDIxNlQxNzAwMDBaIHVzaTlxNGdjMGVsbjJ2NzM5MnRxYjhrNGs0QGc",
```
Comment (iTweeno) [inline src/utils/__tests__/getMapFilteredEventsByDay.test.ts:93]: here aswell

File: src/components/Home/__tests__/Home.test.tsx (hunk 1 above)
Comment (zpawn) [reply src/components/Home/__tests__/Home.test.tsx]: done

File: src/utils/__tests__/getMapFilteredEventsByDay.test.ts (hunk 2 above)
Comment (zpawn) [reply src/utils/__tests__/getMapFilteredEventsByDay.test.ts]: done

**Review by iTweeno — APPROVED** [review]

**Pull request closed without merging**